### PR TITLE
Nd user profile

### PR DESCRIPTION
**Pull request by SaraDawner2000**
Complete Industrial Photogram Part 4

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -22,10 +22,11 @@ def edit
   # POST /comments or /comments.json
   def create
     @comment = Comment.new(comment_params)
+    @comment.author = current_user
 
     respond_to do |format|
       if @comment.save
-        format.html { redirect_to comment_url(@comment), notice: "Comment was successfully created." }
+        format.html { redirect_back fallback_location: root_path, notice: "Comment was successfully created." }
```
Comment (YoungSong99) [inline app/controllers/comments_controller.rb:29]: Nice redirection using fallback_location!

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            </li>
+          <% end %>
+        </ul>
+        <div class="card-body">
+          <%= render "comments/form", comment: photo.comments.build %>
+        </div>
+      </div>
+    </div>
+  </div>
+<% end %>
```
Comment (YoungSong99) [inline app/views/users/show.html.erb:34]: Nice looking card using bootstrap 💯

File: lib/tasks/dev.rake [hunk 3]
```diff
@@ -49,7 +49,7 @@ task sample_data: :environment do
   photos.each do |photo|
     rand(5).times do
       Comment.create(
-        body: Faker::Quote.fortune_cookie,
+        body: Faker::Quote.yoda,
```
Comment (YoungSong99) [inline lib/tasks/dev.rake:52]: I'm curious to try the quote from yoda!

**Review by YoungSong99 — APPROVED** [review]
Your attention to detail really shines through in the clean code and the way you've used data to make it seem like we're using it in the real world. I love the cat pictures!  🥇

File: app/views/users/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        <div class="card-body">
+          <h5 class="card-title"><%= photo.owner.username %></h5>
+          <p class="card-text"><%= photo.caption %></p>
+        </div>
+        <ul class="list-group list-group-flush">
+          <% photo.comments.each do |comment| %>
+            <li class="list-group-item">
+              <div class="d-flex">
+                <div class="flex-shrink-0">
+                  <img src="..." alt="...">
```
Comment (JoseMDO) [inline app/views/users/show.html.erb:18]: We could get rid of this if it is not being used, or we can add default values :)

File: app/views/comments/_form.html.erb [hunk 5]
```diff
@@ -11,22 +11,13 @@
...
-
-  <div>
-    <%= form.label :photo_id, style: "display: block" %>
-    <%= form.text_field :photo_id %>
-  </div>
-
-  <div>
-    <%= form.label :body, style: "display: block" %>
-    <%= form.text_area :body %>
+  <%= form.hidden_field :photo_id %>
```
Comment (JoseMDO) [inline app/views/comments/_form.html.erb:14]: Nice job using hidden inputs!

**Review by JoseMDO — APPROVED** [review]
Everything looks great!

**Pull request merged**